### ISSUE TITLE
Export failed - ffmpeg exited with code: 1

### ISSUE THREAD

**Issue by frozenpandaman** [wulkano/Kap#849]
**macOS version:** 10.14.6
**Kap version:** 3.2.2, installed via Homebrew cask

I can't seem to save/export a specific captured video/recording although it plays fine in the "New Recording" Kap window (where is this saved in cache/temporary storage? perhaps I can manually back up the vile via there?) As the recording is exporting, a file appears on disk and grows in size until getting automatically deleted after it reaches roughly 3.7 GB and Kap returns an error message. I'm guessing it may be because the file is too large/long (around 40 minutes).

I don't think this is a duplicate of either #681 or #474 as this error happens when attempting to export a video as an MP4, or simply via Cmd+S to save the captured video as-is, not as a WebM as was specifically mentioned in those two previous issues. That said, it _also_ happens when trying to export as a WebM, and no matter whether I choose "Save to Disk" or "Open With" (mpv, QuickTime Player, etc.)

Full error message is below:
```
ffmpeg exited with code: 1

ffmpeg version N-92718-g092cb17983-tessus https://evermeet.cx/ffmpeg/ Copyright (c) 2000-2018 the FFmpeg developers
  built with Apple LLVM version 10.0.0 (clang-1000.11.45.5)
  configuration: --cc=/usr/bin/clang --prefix=/opt/ffmpeg --extra-version=tessus --enable-avisynth --enable-fontconfig --enable-gpl --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libfreetype --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libmysofa --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenh264 --enable-libopenjpeg --enable-libopus --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzimg --enable-libzmq --enable-libzvbi --enable-version3 --pkg-config-flags=--s
```

**Comment by frozenpandaman** [comment]
As a workaround, thanks to https://github.com/wulkano/kap/issues/606#issuecomment-453733098, I was able to find the original recording in /private/var/folders and copy it from there.

**Comment by frozenpandaman** [comment]
This happens just when trying to Cmd+S save the video as well, if the video is long/large enough. The video in /private/far/folders was around 2.3 GB and unable to be saved from Kap – I had to manually copy it out of that temp folder. @skllcrn, I think you should be able to reproduce this if you have a recording that's over 20 minutes in length or so and try to save or export it elsewhere.

**Comment by shaodahong** [comment]
same issue

**Comment by shaodahong** [comment]
macOS version: 10.15.6
Kap version: 3.2.2, installed via Homebrew cask

**Comment by karaggeorge** [comment]
@shaodahong @frozenpandaman 

This is specific, but do you remember if you changed any options in the editor before exporting (dimensions/crop time)? Also if you still have the recording or remember, was one of the dimensions odd?

**Comment by frozenpandaman** [comment]
@karaggeorge I didn't change any options – just went directly to export. No clue if one of the dimensions was an odd number or not, sorry! (I just selected an inexact portion of my screen, so 50/50 chance 😉)

**Comment by karaggeorge** [comment]
This might have been addressed here: https://github.com/wulkano/Kap/pull/902/files#diff-90d9e913655ab0b38adfba9f90394bdcR147

I'm not 100% sure if that's what was causing this issue, but I did catch that if you don't change any options, we avoid cropping. But then we also don't make dimensions even, which fails some exports. I guess we can wait until that is released and see if this happens again

**Comment by shaodahong** [comment]
no, and reinstall, it still has

**Comment by shaodahong** [comment]
Is it related to version FFmpeg?

**Comment by shaodahong** [comment]
I'm sure it was good when I used it before, but I both install FFmpeg use `brew`，`kap` use local FFmpeg bin?

**Comment by shaodahong** [comment]
good news, i just change size, it completed, But it never happened before

**Comment by karaggeorge** [comment]
If you changed the size and it worked, it could be caused by the issue I mentioned above. So when that PR is released we'll keep an eye one this and see if it happens again

**Comment by karaggeorge** [comment]
This should be fixed in the new release 3.3.0

If anyone experiences this with 3.3.0 and later, we can re-open
